client/checkout/classic/upe.js

```diff
@@ -743,15 +755,6 @@ jQuery( function ( $ ) {
 
 	// Handle the add payment method form for WooCommerce Payments.
 	$( 'form#add_payment_method' ).on( 'submit', function () {
-		if (
-			'woocommerce_payments' !==
-			$(
-				"#add_payment_method input:checked[name='payment_method']"
```

Comment:
Are we removing this because in http://localhost:8082/my-account/add-payment-method/, it will no longer be only "Credit Card/debit card"?

![image](https://user-images.githubusercontent.com/572862/182942810-084c68bd-f8e0-45f0-9843-b2cfd6a2797a.png)

Do you know why we needed to `return;` early before?